.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/telescope
+    docker:
+      - image: circleci/node:10
+      - image: circleci/redis:5 
```

Comment:
Is there a specific reason we are going for `redis:5`?
As per the [docs](https://circleci.com/docs/2.0/circleci-images/#redis), `latest` tag is also available.
We should go for `latest` tag, saving us from depending on a particular variant of docker image of redis.

Reply:
I added latest for both node and redis. Thanks for pointing that out.